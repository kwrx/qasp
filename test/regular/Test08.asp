%@forall
a | b.
%@exists
c | d.
%@constraint
:- not c.