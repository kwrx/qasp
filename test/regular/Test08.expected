QASP 1.0

{a,c}
{b,c}
